Dart/ReadMe.txt

```diff
@@ -25,7 +25,8 @@ Prerequisites:
 
 4. Open intellij-community project, compile it.
    Open File | Project Structure | Modules | [+] | Import Module, select intellij-plugins/Dart/Dart-community.iml.
-   Open community-main module, add dependency on Dart module.
+   In the list, select community-main then click [+] at the bottom of the list to
+   add a dependency on the Dart-community module.
```

Comment:
There are a lot of [+] buttons. I'd specify that it is in the Dependencies tab. Different location on Mac (bottom) and Windows (right).
How about this:
In the same Project Structure dialog open Dependencies tab of the community-main module, click [+] to add a module dependency on the Dart-community module.